.github/workflows/vscode-marketplace.yml

```diff
@@ -0,0 +1,37 @@
+name: publish to VSCode marketplace
+
+on:
+  workflow_dispatch:
+    inputs:
+      URL:
+        description: 'input the URL of the VSIX.'
+        required: true
+        default: ''
+
+jobs:
+  publish-to-vscode-marketplace:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: 14
+
+      - name: Install VSCE command
+        run: |
+          npm install vsce -g
+
+      - name: Download VISX fils
+        uses: nick-invision/retry@v2
+        with: 
+          timeout_minutes: 10
+          max_attempts: 10
+          retry_on: error
+          shell: bash
+          command: |
+            wget ${{ github.event.inputs.URL }} 
+
+      - name: release to VSCode marketplace
+        run: |
+          vsce publish --pat ${{ secrets.VSCE_PAT }} --packagePath *.vsix
```

Comment:
If an issue occurs during the step **Download VISX files**, it will retry to download it again. Then there will be two downloaded vsix packages:
1. ms-teams-vscode-extension-2.0.1.vsix
2. ms-teams-vscode-extension-2.0.1.vsix.1

`*.vsix` in the **release to VSCode marketplace** step will point to the first package, however, the right package should be the second one (downloaded by the second round) since the first package is broken.

Suggestions:
1. We need to reset the status (clean up the broken downloaded files) before retrying it again.
2. It would be better to use the built-in retry parameter of `wget` to take care of this case.